src/common/utils/time-utils.test.ts

```diff
@@ -7,13 +7,14 @@ import {
   dateToUTC,
   addTime,
   subtractTime,
+  getStartOf,
+  getTimeDiff,
 } from 'common/utils/time-utils';
 
 // Date object constructor takes a zero-indexed month number.
 
-const testDate = new Date(2020, 2, 1);
-
 describe('time formatting', () => {
+  const testDate = new Date(2020, 2, 1);
   test('Date in ISO format', () => {
     expect(formatDateTime(testDate, TimeFormat.YYYY_MM_DD)).toBe('2020-03-01');
```

Comment:
just looked at the `TimeFormat` enum--maybe naming it `DateFormat` would make more sense?